test/benchmarks/perftest.pl

```diff
@@ -491,14 +492,16 @@ ()
     return %result;
 }
 
-sub runtest($$)
+sub runtest
 {
     my $testcasename = shift;
     my $variant = shift;
 
     my %results = ("time", -1, "score", -1);
 
-    $other_switches .= " " . @testlistSwitches{$testcasename};
+    #TODO investigate what was meant to go here, other than adding whitespace on the end
+    #This is a hash slice syntax on an array which is empty, so makes no sense at all
+    #$other_switches .= " " . @testlistSwitches{$testcasename};
```

Comment:
Anyway I'll assume the space was necessary unless shown otherwise (all parameters to scripts need to have spaces between them, so we wouldn't want to cut a necessary space out). As for the evaluation of that expression, without actually running an example, \*shrug\*